tests/system/test_gbq.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 import pandas
 import pandas.api.types
-import pandas.util.testing as tm
+import pandas.testing as tm
```

Comment:
What does "tm" stand for?

Reply:
Maybe "test module"? It's one of the leftovers from the pandas repo. e.g.

https://github.com/pandas-dev/pandas/blob/059c8bac51e47d6eaaa3e36d6a293a22312925e6/pandas/tests/arrays/categorical/test_sorting.py#L8

Reply:
Ah, it's test module, now I remember.

(I'm generally not a fan of possibly-cryptic abbreviations :slightly_smiling_face: )